domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityComplexFieldACLValidatorImpl.java

```diff
@@ -118,17 +118,17 @@ private boolean isAChild(String parentCode, String childCode) {
     private boolean isMissingInComplexACLs(List<ComplexFieldACLEntity> complexFieldACLEntities,
                                            String userRole, String parentCode) {
         return complexFieldACLEntities.stream()
-            .noneMatch(entity -> (entity.getUserRole() != null
-                && entity.getUserRole().getReference().equalsIgnoreCase(userRole))
+            .noneMatch(entity -> (entity.getAccessProfile() != null
+                && entity.getAccessProfile().getReference().equalsIgnoreCase(userRole))
                 && parentCode.equals(entity.getListElementCode())
             );
     }
 
     private void validateCrudAgainstCaseFieldParent(CaseFieldEntity caseField,
                                                     CaseFieldEntityValidationContext caseFieldEntityValidationContext,
                                                     ValidationResult validationResult, ComplexFieldACLEntity entity) {
-        String userRole = entity.getUserRole() != null ? entity.getUserRole().getReference() : "";
-        final Optional<CaseFieldACLEntity> caseFieldACLByRole = caseField.getCaseFieldACLByRole(userRole);
+        String accessProfile = entity.getAccessProfile() != null ? entity.getAccessProfile().getReference() : "";
+        final Optional<CaseFieldACLEntity> caseFieldACLByRole = caseField.getCaseFieldACLByAccessProfile(accessProfile);
```

Comment:
Update `caseFieldACLByRole` variable name

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f